mkdocs/docs/contributing.md

```diff
@@ -234,7 +234,7 @@ Deprecated in 0.1.0, will be removed in 0.2.0. The old_property is deprecated. P
 
 For the type annotation the types from the `Typing` package are used.
 
-PyIceberg offers support from Python 3.9 onwards, we can't use the [type hints from the standard collections](https://peps.python.org/pep-0585/).
+PyIceberg offers support from Python 3.10 onwards, we can't use the [type hints from the standard collections](https://peps.python.org/pep-0585/).
```

Comment:
i think we can revert this change. 

https://peps.python.org/pep-0585/ is supported by 3.9

Reply:
ah looks like this was find/replaced when we removed 3.8

https://github.com/apache/iceberg-python/commit/857abd09d5cd547b357b6eb66dd327fecec4c605#diff-c0ad53549c03c8e66a0e3574698ce24dfa722b54ca7a2fdfcb8d62ff0343ab73R205-R206

Reply:
i think we can remove this paragraph entirely then